### ISSUE TITLE
[feature request] loadUpdateData only insert

### ISSUE THREAD

**Issue by yelhouti** [liquibase/liquibase#1101]
## Description

The same way we have an onlyUpdate option for loadUpdateData could we have an onlyInsert that doesn't edit existing entries, and only add new ones:

purpose:
Init database with defaults required to run the app, and allow to add other defaults later without overriding changed values or the need for new files...

example:
default content for emails, user can change default email, add new emails as app becomes more complex by adding lines to csv

**Comment by molivasdat** [comment]
Hi @yelhouti .  Thank you for requesting this new feature.  If I understand the request successfully, you are requesting that another attribute is added to the loadUpdateData parameter (https://www.liquibase.org/documentation/changes/load_update_data.html)  called something like onlyInserts, that checks for the existence of the data and if it exists in the table ignore it and only add if it does not already exist.

**Comment by yelhouti** [comment]
@molivasdat yes indeed, it would do that check using the primary key of the table

**Comment by yelhouti** [comment]
Is there a way to add bounties on issues like, I am willing to pay few bucks for this to be implemented since i don't have time myself

**Comment by yelhouti** [comment]
I played with this a little bit, and I am not sure if the param is better added load_update_data or load_data or even adding a separate tag load_if_not_exists . Multiple database has different requirements in order to implement the feature.
Mysql/Mariadb has : https://mariadb.com/kb/en/insert-ignore/ `INSET IGNORE...`
Postgres has: https://www.postgresql.org/docs/current/sql-insert.html#SQL-ON-CONFLICT `INSERT ... ON CONFLICT IGNORE`
While other database have a more complex syntax that might require the conflict constraint name...
I think After all this that the quickest approach is implementing on top of loadData.

**Comment by exijn** [comment]
Hi, I'd be interested in working on this issue if the feature is still desired.

My initial plan is to add an `onlyInsert` option to `loadUpdateData`, make it mutually exclusive with `onlyUpdate`, preserve the existing default behavior, and add tests verifying that existing rows remain unchanged while new rows are inserted.

Since this issue has been open for some time, could a maintainer confirm that this API and behavior would still be accepted before I begin the implementation?
